pkg/apis/infinispan/v1/infinispan_types.go

```diff
@@ -141,7 +141,7 @@ type InfinispanStatus struct {
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// +optional
 	Conditions              []InfinispanCondition `json:"conditions,omitempty"`
-	StatefulSetName         string                `json:"statefulSetName"`
+	StatefulSetName         string                `json:"statefulSetName,optional,omitempty"`
```

Comment:
@rigazilla please pay attention, that this `StatefulSetName` properties not actual at master and should be ported.